stylesheets/fonts.css

```diff
@@ -0,0 +1,13 @@
+/* fredericka-the-great-regular - latin-ext_latin */
+@font-face {
+  font-family: 'Fredericka the Great';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../fonts/fredericka-the-great-v10-latin-ext_latin-regular.eot'); /* IE9 Compat Modes */
+  src: local(''),
+       url('fonts/fredericka-the-great-v10-latin-ext_latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('fonts/fredericka-the-great-v10-latin-ext_latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
+       url('fonts/fredericka-the-great-v10-latin-ext_latin-regular.woff') format('woff'), /* Modern Browsers */
+       url('fonts/fredericka-the-great-v10-latin-ext_latin-regular.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('fonts/fredericka-the-great-v10-latin-ext_latin-regular.svg#FrederickatheGreat') format('svg'); /* Legacy iOS */
```

Comment:
Shouldn't these be `../fonts/<path>` ?

Reply:
Ooh sorry. Can you please tell the difference among the two, i.e. using the ../ before the path and not using them before the path

Reply:
`../` refers to the parent directory of the current directory. Lets say you have a file structure like `/parent/child/grandchild`. If you are in the `grandchild` directory, `./` refers to the `grandchild` directory, `../` refers to the `child` directory, `../../` refers to the `parent directory`, and so on.

In this case, since the `fonts.css` file is within `stylesheet` directory, using `fonts/<path>` will try to look for `fonts` folder in the `stylesheets` directory. However we know that we need to go one level above the current directory to find `fonts` folder. So we use `../fonts/<path>`.

I hope this makes it clear.

Reply:
Thanks for the help, I got it.